build-support/docker/centos6/Dockerfile

```diff
@@ -6,16 +6,27 @@ FROM centos:6
 
 # Install python 2.7.13, (more) modern gcc, and a JDK.
 RUN yum -y update
+# TODO: figure out why this needs to be installed first for /usr/bin/scl to work!
 RUN yum install -y centos-release-scl
-RUN yum install -y \
-  devtoolset-7-gcc \
-  devtoolset-7-gcc-c++ \
+RUN yum install -y devtoolset-7-gcc{,-c++} \
   git \
   java-1.8.0-openjdk-devel \
   python27 \
```

Comment:
Do we need the yum-installed python27 now, or can we install it via pyenv too?

Reply:
For consistency, I think pyenv makes sense, albeit not necessary. Also allows us to use Python 2.7.15.